ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/ScanConfigPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+
+import { policiesBasePath } from 'routePaths';
+import NotFoundMessage from 'Components/NotFoundMessage';
+import PageTitle from 'Components/PageTitle';
+import { getScanConfig, ScanConfig } from 'services/ComplianceEnhancedService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { BasePageAction } from 'utils/queryStringUtils';
+
+import { initialScanConfig } from './scanConfigs.utils';
+// import ScanConfigWizard from './Wizard/ScanConfigWizard';
+
+// type WizardScanConfigState = {
+//     wizardScanConfig: ClientScanConfig;
+// };
+
+type ScanConfigPageProps = {
+    // eslint-disable-next-line react/no-unused-prop-types
+    hasWriteAccessForCompliance: boolean;
+    pageAction?: BasePageAction;
+    scanConfigId?: string;
+};
+
+function ScanConfigPage({
+    // TODO: for creating new scan schedules
+    // hasWriteAccessForCompliance,
+    pageAction,
+    scanConfigId,
+}: ScanConfigPageProps): ReactElement {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [scanConfig, setScanConfig] = useState<ScanConfig>();
+    //     pageAction === 'generate' && wizardScanConfig
+    //         ? getClientWizardScanConfig(wizardScanConfig)
+    //         : initialScanConfig
+    const [scanConfigError, setScanConfigError] = useState<ReactElement | null>(null);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        setScanConfigError(null);
+        if (scanConfigId) {
+            // action is 'clone' or 'edit' or undefined
```

Comment:
This comment reminds to suggest as you fill in the implementation, some containers do not ignore clone, create, or edit action types from page address query string if use role does not have write access.

Collections is the best, although found one possible loophole while I searched for this:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Collections/CollectionForm.tsx#L219

Reply:
Did you mean "... some containers do ~not~ **ignore** clone, create, or edit action types from page address query string **if use role does not have write access**." ?

Reply:
Good question.

I did mean **not** ignore in negative sense that in come containers, page address provides a back door to clone/create/edit **but** components need to do the following instead:
* ignore bogus create action: render table
* ignore bogus clone or edit actions: render page in view mode